assets/helpers/tracking/googleTagManager.js

```diff
@@ -0,0 +1,26 @@
+import uuidv4 from 'uuid';
+import { forCountry } from '../internationalisation/currency';
+import { get as getCookie } from '../cookie';
+
+// Generates a uuid to anonymously identify this user in this session.
+// We need this to prevent page refreshes on conversion pages being
+// treated as new conversions
+export function getOrderId() {
+  let orderId = sessionStorage.getItem('orderId');
+  if (orderId === null) {
+    orderId = uuidv4();
+    sessionStorage.setItem('orderId', orderId);
+  }
+  return orderId;
+}
+
+export function getCurrency() {
+  const country = getCookie('GU_country');
```

Comment:
Is it not possible to reuse the `detect` method from country helper? It is [here](https://github.com/guardian/support-frontend/blob/master/assets/helpers/internationalisation/country.js#L138)

Reply:
👍